drivers/sensor/ti/ina23x/ina230.c

```diff
@@ -185,14 +231,19 @@ static int ina230_calibrate(const struct device *dev)
 static int ina230_init(const struct device *dev)
 {
 	const struct ina230_config *const config = dev->config;
+	uint16_t config_raw = config->config;
 	int ret;
 
 	if (!device_is_ready(config->bus.bus)) {
 		LOG_ERR("I2C bus %s is not ready", config->bus.bus->name);
 		return -ENODEV;
 	}
 
-	ret = ina23x_reg_write(&config->bus, INA230_REG_CONFIG, config->config);
+	if (ina230_is_high_precision(config)) {
+		config_raw |= INA236_CONFIG_ADC_RANGE;
+	}
+
+	ret = ina23x_reg_write(&config->bus, INA230_REG_CONFIG, config_raw);
```

Comment:
This seems like an unnecessary change for this commit. I think it should either be reverted or have it's own commit with a change reason.

Reply:
Do you mean that the support for high-precision should be added in a separate commit?

Reply:
I mean adding a variable to hold the value of config->config. There's no functional change here, just coding style.

Reply:
I think I do not understand you, but I reverted the `config_raw` usage and used `config->config` instead. Enabling the `high-precision` mode is done when creating an instance of the device.